data.js

```diff
@@ -1,57 +1,51 @@
-let Title;
-let Author;
-let bookList = [];
-
-const getValues = () => {
-  Title = document.querySelector('#title').value;
-  Author = document.querySelector('#author').value;
-  document.querySelector('#title').value = '';
-  document.querySelector('#author').value = '';
-};
-
-const addBook = () => {
-  document.getElementById('dynamic').innerHTML = '';
-  bookList.forEach((book, index) => {
-    document.getElementById('dynamic').innerHTML += `
-    <div class="books">
-      <p class="title">${book.title}</p>
-      <p class="authir">${book.author}</p>
-      <button class="remove" onClick = 'removeItem(${index})'>Remove</button>
-      <hr>
-    </div>`;
-  });
-};
-
-if (localStorage.getItem('Books')) {
-  bookList = JSON.parse(localStorage.getItem('Books'));
-  addBook();
+/* eslint-disable-next-line max-classes-per-file */
+class Book {
+  constructor(Title, Author) {
+    this.Title = Title;
+    this.Author = Author;
+  }
 }
 
-const storeBook = () => {
-  getValues();
-  const data = {
-    title: Title,
-    author: Author,
-  };
-  if (data.title !== '' && data.author !== '') {
-    bookList.push(data);
+class BookL {
+  bookList = [];
+
+  add() {
+    const Title = document.querySelector('#title').value;
+    const Author = document.querySelector('#author').value;
+
+    if (Title !== '' && Author !== '') {
+      const book = new Book(Title, Author);
+      this.bookList.push(book);
+      localStorage.setItem('Books', JSON.stringify(this.bookList));
+      this.render();
+    }
+    document.querySelector('#title').value = '';
+    document.querySelector('#author').value = '';
   }
-  Title = '';
-  Author = '';
-  localStorage.setItem('Books', JSON.stringify(bookList));
-  addBook();
-};
-
-const btnAdd = document.getElementById('btn');
 
-btnAdd.addEventListener('click', storeBook);
-
-const removeItem = (index) => {
-  bookList.splice(index, 1);
-  localStorage.setItem('Books', JSON.stringify(bookList));
-  addBook();
-};
+  render() {
+    document.getElementById('dynamic').innerHTML = '';
+    this.bookList = JSON.parse(localStorage.getItem('Books'));
+    this.bookList.forEach((book, index) => {
```

Comment:
- [ ] When the page loads initially, it does not have any `bookList` data in the local storage. As a result, the `forEach` method throws a console error when the page loads initially. Kindly handle this error. :pray: 

![Screenshot from 2021-12-01 14-44-32](https://user-images.githubusercontent.com/64894927/144228996-e553f421-df32-4104-81db-41adee9a0991.png)